tests/core/test_asr_metrics.py

```diff
@@ -86,14 +86,15 @@ def get_wer(prediction: str, reference: str):
             )
             return res[0] / res[1]
 
-        self.assertEqual(get_wer('cat', 'cot'), 1.0)
-        self.assertEqual(get_wer('gpu', 'g p u'), 1.0)
-        self.assertEqual(get_wer('g p u', 'gpu'), 3.0)
-        self.assertEqual(get_wer('ducati motorcycle', 'motorcycle'), 1.0)
-        self.assertEqual(get_wer('ducati motorcycle', 'ducuti motorcycle'), 0.5)
-        self.assertEqual(get_wer('a f c', 'a b c'), 1.0 / 3.0)
+        assert get_wer('cat', 'cot') == 1.0
+        assert get_wer('gpu', 'g p u') == 1.0
+        assert get_wer('g p u', 'gpu') == 3.0
+        assert get_wer('ducati motorcycle', 'motorcycle') == 1.0
+        assert get_wer('ducati motorcycle', 'ducuti motorcycle') == 0.5
+        assert get_wer('a f c', 'a b c') == 1.0 / 3.0
 
     @pytest.mark.unit
+    @pytest.mark.pleasefixme
```

Comment:
Can we unmark for now? Let it error out and address it then?

Reply:
right! thanks, forgot about this...

Reply:
done